.travis.yml

```diff
@@ -21,8 +21,10 @@ before_script:
   - ./cc-test-reporter before-build
 
 script:
-  - bundle exec rake code_analysis
-  - bundle exec rspec
+  - echo 'Nothing to run'
+  # # When adding a new cop, uncomment the lines below to analyze/test in the CI
+  # - bundle exec rake code_analysis
```

Comment:
What's the effect of commenting these 2 lines?

Reply:
Nothing really, we currently don't have any custom cop in here so it's only a repository of default config files for rubocop. 

So that's why there's currently no need to run code analysis nor tests, because there isn't any code to test or analyze.